test/unit/src/math/Quaternion.tests.js

```diff
@@ -391,6 +391,18 @@ export default QUnit.module( 'Maths', () => {
 
 		} );
 
+		QUnit.test( "angleTo", ( assert ) => {
+
+			var a = new Quaternion();
+			var b = new Quaternion().setFromEuler( new Euler( 0, Math.PI, 0 ) );
+			var c = new Quaternion().setFromEuler( new Euler( 0, Math.PI * 2, 0 ) );
+
+			assert.ok( a.angleTo( a ) <= eps, "Passed!" );
+			assert.ok( ( a.angleTo( b ) - Math.PI ) <= eps, "Passed!" );
+			assert.ok( ( a.angleTo( c ) - ( Math.PI * 2 ) ) <= eps, "Passed!" );
```

Comment:
Sorry,  but this test is not adequate.  You are only testing the zero quaternion. And your angles are multiples of PI.

Also, `a` and `c` represent the same orientation, but `a.angleTo( c )` is not zero.

Reply:
> Also, a and c represent the same orientation, but a.angleTo( c ) is not zero.

The value is 360 Degrees. I think there is no "correct" results since it's open to interpretation.

Reply:
I would say that is a problem, then, because it means output of this method is not particularly useful as a metric.